### ISSUE TITLE
coredns <-> pods service locality?

### ISSUE THREAD

**Issue by allen-bmf** [miekg/coredns#6296]
hello
I have question coredns
AWS EKS environment. Do coredns also need a pod<->coredns service locality?
If so, is there a way to access the same zone?
Also, if I use istio, how can I configure it?

**Comment by chrisohaver** [comment]
Can you rephrase the question?  I don't understand it.

**Comment by Kyslik** [comment]
I believe you will benefit from reading following docs:
- https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/
- https://coredns.io/plugins/kubernetes/
- https://docs.aws.amazon.com/eks/latest/userguide/managing-coredns.html
